src/engine/ExportQueryExecutionTrees.h

```diff
@@ -157,6 +157,14 @@ class ExportQueryExecutionTrees {
       ad_utility::streams::stream_generator streamGenerator);
 
  private:
+  // Make sure that the offset is not applied again when exporting the
+  // result (it is already applied by the root operation in the query
+  // execution tree). Note that we don't need this for the limit because
+  // applying a fixed limit is idempotent. This only works because the query
```

Comment:
Make sure that in the corresponding place in the query planner there is a comment that tells you to also look at this place, should something ever change there.